docs/codeql/query-help/index.rst

```diff
@@ -0,0 +1,22 @@
+CodeQL query help
+-----------------
+
+View the query help for the queries included in the ``code-scanning``, ``security-extended``, and ``security-and-quality`` query suites for the languages supported by CodeQL.
+
+Each query help article includes: 
+
+- A summary of key metadata for the query.
+- Information about which query suites the query is included in.
+- A link to the query in the `CodeQL repository <https://github.com/gitub/codeql>`__
+- Help information to explain what potential vulnerability the query identifies and a recommendation for how to avoid introducing the problem to your code.
```

Comment:
```suggestion
- A description of the potential vulnerability that the query identifies and a recommendation for how to avoid introducing the problem to your code.
```

Minor tweak